src/Evolution/DiscontinuousGalerkin/Actions/ApplyBoundaryCorrections.hpp

```diff
@@ -573,6 +576,10 @@ void ApplyBoundaryCorrections<Metavariables>::complete_time_step(
             }
             mortar_id_ptr = &mortar_id;
             compute_correction_coupling(mortar_data, mortar_data);
+            // Remove data since it's tagged with the time. We in _might_ be
+            // able to reuse allocations, but this is an optimization after some
+            // profiling.
```

Comment:
Fix english:
- "we in might be" -> "in the future we might be"
- "this is an optimization after some profiling" -> "this optimization should only be done after profiling"?